### PR TITLE
AIK-3168 : Pour sql_injection vulnerability code over to python firewall (including testing)

### PR DESCRIPTION
**Pull request by bitterpanda63**

### CODE REVIEW

File: aikido_firewall/sinks/pymysql.py [hunk 1]
```diff
@@ -28,6 +33,11 @@ def aikido_new_query(_self, sql, unbuffered=False):
         logger.debug("Sql : %s", sql)
         context = get_current_context()
         logger.debug("Context according to MySQL wrapper : %s", context)
+        result = check_context_for_sql_injection(sql, "Test_op", context, MySQL())
+        logger.critical("SQL RESULTS : ")
+        logger.critical(json.dumps(result))
```
Comment (willem-delbare) [inline aikido_firewall/sinks/pymysql.py]: best not to log (potential pii etc)

File: aikido_firewall/vulnerabilities/sql_injection/uinput_occ_safely_encapsulated_test.py [hunk 2]
```diff
@@ -0,0 +1,162 @@
+import pytest
+from aikido_firewall.vulnerabilities.sql_injection.uinput_occ_safely_encapsulated import (
+    uinput_occ_safely_encapsulated,
+)
+
```
Comment (willem-delbare) [inline aikido_firewall/vulnerabilities/sql_injection/uinput_occ_safely_encapsulated_test.py:6]: lets add tests for js-slice

File: aikido_firewall/sinks/pymysql.py (hunk 1 above)
Comment (bitterpanda63) [reply aikido_firewall/sinks/pymysql.py]: Done

File: aikido_firewall/vulnerabilities/sql_injection/uinput_occ_safely_encapsulated_test.py (hunk 2 above)
Comment (bitterpanda63) [reply aikido_firewall/vulnerabilities/sql_injection/uinput_occ_safely_encapsulated_test.py:6]: Sec

Comment (bitterpanda63) [reply aikido_firewall/vulnerabilities/sql_injection/uinput_occ_safely_encapsulated_test.py:6]: Love to AI, it's done

**Pull request merged**